custom/icds_reports/utils/aggregation_helpers/distributed/base.py

```diff
@@ -196,7 +196,10 @@ def aggregate_temp(self, cursor):
 
     def update_table(self, cursor):
         update_queries = self.update_queries()
-        rollup_queries = [self.rollup_query(i) for i in range(4, 0, -1)]
+        if getattr(self, 'rollup_query', None):
```

Comment:
nit: The `None` here is redundant `getattr` defaults to returning none.

Reply:
It looks in python 3 its mendatory.